src/client/contextChannels.ts

```diff
@@ -335,7 +335,7 @@ export class DesktopChannel extends ChannelBase {
     public readonly color: number;
```

Comment:
There's some broken `@link` tags elsewhere in this file - I think `Channel` needs to be `BaseChannel`.

Reply:
Done